jobrunner/cli/flags.py

```diff
@@ -0,0 +1,80 @@
+"""
+Ops utility getting and setting db flags
+"""
+import argparse
+import sqlite3
+import sys
+from dataclasses import dataclass
+
+from jobrunner.lib.database import create_table, get_connection, get_flag, set_flag
+
+
+NOVALUE = object()
+
+
+@dataclass
+class Flag:
+    """Do you have a flaaaaaag?"""
+
+    name: str
+    value: str = None
+
+    @classmethod
+    def from_cli(cls, raw):
+        name, equals, value = raw.partition("=")
+        if value == "":
+            if equals == "=":
+                # flag=, set flag to None
+                value = None
+            else:
+                value = NOVALUE
+        return cls(name, value)
+
+
+def main(flags, create):
+    try:
+        do_flags(flags)
+    except sqlite3.OperationalError as e:
+        if "no such table" in str(e):
+            if create:
+                create_table(get_connection(), "flags")
+                do_flags(flags)
+            else:
+                sys.exit(
+                    "Flags table does not exists. Run command again with --create to create it."
+                )
```

Comment:
Nice :+1:  I think it's the right call to make this an explicit action, but explain what to do in the error message.